docs/HACKING_QUICKSTART.md

```diff
@@ -108,20 +108,28 @@ This is how my projects are laid out:
 
 ```
 ~/my-projects/servo/
-~/my-projects/cocoa-rs/
-~/my-projects/glutin/
+~/my-projects/webrender/webrender
```

Comment:
I *think* that you mean for `~/my-projects/webrender` to be the root of the git repository, and the `webrender` directory inside of it contains `Cargo.toml` for that crate. But that’s not very clear. To make it consistent with the line above (which is `~/my-projects/servo`, not `~/my-projects/servo/ports/servo`), maybe change this line to just `~/my-projects/webrender`?